docs/Using-EthSigner/Azure.md

```diff
@@ -0,0 +1,50 @@
+description: Signing transactions with key stored in Azure Key Vault
+<!--- END of page meta data -->
+
+# Using EthSigner with Azure Key Vault 
+
+EthSigner supports storing the signing key in a [Azure Key Vault](https://azure.microsoft.com/en-au/services/key-vault/). 
+
+!!! caution 
+    EthSigner with Azure Key Vault does not support TLS. If using a remote Azure Key Vault, apply appropriate 
```

Comment:
Is this a thing? I.e. the Azure connection etc is done inside the Azure supplied libraries ... I _assume_ it performs secure connections ... _assume_ only though :/

Reply:
Removed